homeassistant/components/egardia.py

```diff
@@ -0,0 +1,114 @@
+"""
+Interfaces with Egardia/Woonveilig alarm control panel.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/egardia/
+"""
+import logging
+import requests
```

Comment:
Please add a blank line between import groups standard library and 3rd party.

Reply:
Done.